### DISCUSSION TITLE
Add osu! option to allow opening multiple collections at once

### DISCUSSION THREAD

**Discussion by Ceglaa** [ppy/osu#25012] [Ideas]
Also make the F2 random song be drawed from multiple opened collections

**Comment by Rick-Bossen** [comment]
Would also really like for this to be a thing, adding to this, one thing that seems kinda confusing to me is the way that the collections menu works right now. Right now what happens when you open it is that you have the + buttons to add the current selected map to the collection (and a - to remove it if it's already in a collection).

Not sure exactly why, but it would seem feel more logical to me to have these + and - buttons function as a way to add or remove collections from the current selection of collections, allowing you to have multiple active at once. Adding and removing maps should only happen in the right click context menu IMO.

Come to think about it, I think the reason for this feeling kinda inconsistent is because I'm expecting to make modifications to the use of entire collection in the collection dropdown, and only make changes to maps in a collection from the context menu of that specific map, if that makes sense.

**Comment by peppy** [comment]
This should be fixed with the song select redesign, as it will behave like stable.
